packages/web-frontend/src/containers/MapOverlayBar/TitleAndDatePicker.js

```diff
@@ -91,12 +91,6 @@ export const TitleAndDatePickerComponent = ({
   setPinnedOverlay,
   maxSelectedOverlays,
 }) => {
-  const [isSwitchedOn, setIsSwitchedOn] = useState(true);
-
```

Comment:
Didn't notice `isSwitchedOn` is overlapping with `displayedMapOverlays`. Need to remove.